release-notes/unreleased/IGAPP-699-webview-crashing.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-699
+show_in_stores: true
+platforms:
+  - android
+en: Frequent app crashes fixed.
```

Comment:
Maybe we could express this more positive. Like "increased app stability"? When I would read this in a store I maybe would not download the app, assuming it is very buggy.